modules/core/src/main/scala/org/tessellation/http/p2p/P2PClient.scala

```diff
@@ -23,5 +23,5 @@ sealed abstract class P2PClient[F[_]] private (
   val cluster: ClusterClient[F],
   val gossip: GossipClient[F],
   val node: NodeClient[F],
-  val l0GlobalSnapshotClient: L0GlobalSnapshotClient[F]
+  val l0GlobalSnapshot: L0GlobalSnapshotClient[F]
```

Comment:
Out of curiosity, why the rename?

Reply:
@marcinwadon requested it to keep it in line with the other clients.

Reply:
imo, there should be no suffix `Client` (see other names).